src/style.css

```diff
@@ -0,0 +1,36 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
```

Comment:
no need to reset margins and padding for all selector.
reset only for elements where it is necessary